docs/pages/off-canvas.md

```diff
@@ -109,39 +186,47 @@ A design can have two menus: one on the left, and one on the right. Be sure that
 
 ---
 
-## Title Bar
+## Off-canvas Behavior
 
-If you need a simple title bar to toggle the off-canvas, `.title-bar` is here to help. It supports left- and right-aligned sections.
+You can switch the default behavior of the off-canvas from pushing the page over as it open to overlaping the page by adding the `.is-transition-overlap` to the `.off-canvas`. The default option is `.is-transition-push`.
```

Comment:
Think this needs revising a little. The transition is actually applied by adding the option `data-transition="push|overlap` to a panel (push being the default). The class is added depending on what option they pick - I'm not sure the user needs to know that the class is added, thats just part of the styling that makes it work?